helper/send-mail.helper.js

```diff
@@ -0,0 +1,49 @@
+const nodemailer = require("nodemailer");
+const { google } = require("googleapis");
+require("dotenv").config();
```

Comment:
name of the file should be `mail.helper.js`.

Reply:
Sure sir I will be make this changes in next feature branch

Reply:
I will make the changes in the same branch and raise another pr